python/lsst/ip/isr/isrTask.py

```diff
@@ -619,7 +625,7 @@ def getIsrExposure(self, dataRef, datasetType, immediate=True):
         return exp
 
     def saturationDetection(self, exposure, amp):
-        """!Detect saturated pixels and mask them using mask plane "SAT", in place
+        """!Detect saturated pixels and mask them using mask plane config.saturatedMaskName, in place
```

Comment:
I'm not at all convinced this should even be a config param, but as it stands the old docs were wrong.